wiki/commands.md

```diff
@@ -0,0 +1,153 @@
+
+All right. First, get situated on the server.
+
+```bash
+# shell into the server
+ssh ubuntu@ocd.datamade.us
+
+# visit the scrapers directory, and launch the correct virtual environment
+sudo su - datamade
+cd scrapers-us-municipal
+workon opencivicdata
+```
+
+Then, run the appropriate command.
+
+```bash
+# scrape all recently updated data
+pupa update lametro
+
+# scrape all recently updated data, but used cached pages
+# avoid fastmode for larger scrapes, since you risk scraping out-of-date data
+# instead, use the `rpm` option (see below)
+pupa update lametro --fastmode
+
+# scrape all recently updated data, and move as quickly as possible (but do not use cache)
+pupa update lametro --rpm=0 
+```
+
+```bash
+# scrape bills updated in the last 28 days
+# https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/bills.py#L97
+pupa update lametro bills
+
+# scrape all bills
+pupa update lametro bills window=0
+
+# scrape bills updated in the last 7 days, using cached pages
+pupa update lametro bills window=7 --fastmode
+```
+
+```bash
+# scrape all events
+# https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/events.py#L139
+pupa update lametro events
+
+# scrape events updated in the last 7 days
+pupa update lametro events window=7
+```
+
+```bash
+# scrape all people
+# the people scraper does not have a "window" argument
+# but instead determines which people to update by looking at those visible on the web interface
+pupa update lametro people
+```
+
+## Importing data to Councilmatic
+
+`django-councilmatic` comes with a lengthy, sometimes abstruse management command: `import_data`. The [LA Metro Councilmatic README offers some details on it](https://github.com/datamade/la-metro-councilmatic#import-data), and anyone who has the app running locally already knows something about `import_data`. 
+
+**Note!** On the server, [the cron for the production site](https://github.com/datamade/la-metro-councilmatic/blob/master/scripts/lametro-crontasks) executes this command four times per hour. You may need to manually intervene, however. As with the scraper, consider turning off the crons or executing the command during the 15-minute window between cron processes: usually the import runs quickly, so running the command between cronjobs often works best. 
+
+```bash
+# shell into the server
+ssh ubuntu@lametro.datamade.us
+
+# visit the production directory, and launch the correct virtual environment
+sudo su - datamade
+cd lametro
+workon lametro
+
+# run the command!
+python manage.py import_data
```

Comment:
i think it might be worth adding the bit to append the output to our log files for each of these commands, in case the reader doesn't make it all the way to the bottom. would you mind moving the logging section up, and adding `>> /the/appropriate/logfile.txt` to each of the import pipeline commands in this section, for easy copy/paste-ability?